exec/csnex/Exceptions.cs

```diff
@@ -16,6 +23,21 @@ public class NeonException: ApplicationException
 
         public NeonException(string message, System.Exception innerException) : base(message, innerException) {
         }
+
+        // Satisfy Warning CA2240 to implement a GetObjectData() to our custom exception type.
+        [SecurityPermission(SecurityAction.LinkDemand, Flags = SecurityPermissionFlag.SerializationFormatter)]
+        public override void GetObjectData(SerializationInfo info, StreamingContext context)
+        {
+            if (info == null) {
+                throw new ArgumentNullException("info");
+            }
+            info.AddValue("NeonException", Name);
+            info.AddValue("NeonInfo", info);
```

Comment:
Is this right? You seem to be adding `info` to itself, which is likely to end badly. Is this supposed to add `Info` instead?

Reply:
Ah.  Yes, this is wrong.  Fixed.